sound/soc/sof/ipc4-topology.c

```diff
@@ -173,6 +173,40 @@ static void sof_ipc4_dbg_audio_format(struct device *dev,
 	}
 }
 
+static struct sof_ipc4_control_data
+*sof_ipc4_get_widget_init_byte_kcontrol(struct snd_sof_widget *swidget)
```

Comment:
I can't figure out if this is a code move to make the code more readable - as in zero new functionality - or if there are indeed changes. If this is just a code move/refactoring, then make it clear in the commit message.